graphql_compiler/compiler/emit_sql.py

```diff
@@ -1014,10 +1014,13 @@ def recurse(self, vertex_field: str, depth: int) -> None:
         used_columns = sorted(self._used_columns[self._current_location.query_path])
 
         # The base of the recursive CTE selects all needed columns and sets the depth to 0
```

Comment:
This comment might need expanding / updating, once you've gotten this working.

Reply:
Explained a few lines below. See if the comment needs more clarity/context.